### PR TITLE
SEN-91 - Inventory Display

### PR DESCRIPTION
**Pull request by JonathanYcWang**

### CODE REVIEW

File: app/src/components/InventoryItem.vue [hunk 1]
```diff
@@ -1,23 +1,53 @@
 
 <template>
-  <div>
+  <!-- <div>
```
Comment (NElnour) [inline app/src/components/InventoryItem.vue]: is the commented markup still needed?

File: app/store/index.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import PapaParse from "papaparse";
 
-Vue.use(Vuex)
+import axios from 'axios'
```
Comment (NElnour) [inline app/store/index.js]: do we still need axios/vue-axios?

File: app/store/index.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import PapaParse from "papaparse";
+import axios from 'axios'
+import { vuexfireMutations, firestoreAction } from 'vuexfire'
+import db from "../src/plugins/database"
 
-Vue.use(Vuex)
+
+Vue.use(Vuex, axios, vuexfireMutations)
```
Comment (NElnour) [inline app/store/index.js]: axios

Comment (JonathanYcWang) [reply app/store/index.js]: we still need to keep axios in there to send https request. That is seperate from the web sockets

**Pull request merged**